features/step_definitions/general.steps.rb

```diff
@@ -0,0 +1,55 @@
+When(/^I visit root_path$/) do
```

Comment:
Why not reuse the following step

```ruby
Given(/^I visit the "([^"]*)" page$/) do |page_name|
  visit page_path(page_name)
end
```
like this

`Given I visit the "Home" page`